project_tier/exercises/templates/exercises/exercise_index_page.html

```diff
@@ -71,6 +71,9 @@ <h6>Protocol</h6>
       {% else %}
         <p class="filter-text">{{ results.shown }} exercise{{ results.shown|pluralize}} found.</p>
       {% endif %}
+      <div class="notes">
+        {{ self.notes|safe }}
```

Comment:
The filter `self.notes|richtext` is what you're looking for. I'm also gonna wrap this whole thing in a conditional so the div doesn't get rendered if it's empty.